develop/dev-guide-hybrid-oltp-and-olap-queries.md

```diff
@@ -258,7 +258,18 @@ For more information about how TiDB chooses to use TiFlash, see [Use TiDB to rea
 
 ## Read more
 
+<CustomContent platform="tidb">
+
 - [Quick Start with HTAP](/quick-start-with-htap.md)
 - [Explore HTAP](/explore-htap.md)
+
+</CustomContent>
+
+<CustomContent platform="tidb-cloud">
+
+- [TiDB Cloud HTAP Quick Start](/tidb-cloud/tidb-cloud-htap-quickstart.md)
```

Comment:
Will it render the same unsorted list as the above items?

Reply:
yes